conduit/pipeline.go

```diff
@@ -387,18 +412,21 @@ func (p *pipelineImpl) Start() {
 						if err != nil {
 							p.logger.Errorf("%v", err)
 							p.setError(err)
+							retry++
 							goto pipelineRun
 						}
 					}
-					importTime := time.Since(start)
+					metrics.ExporterTimeSeconds.Observe(time.Since(exporterStart).Seconds())
 					// Ignore round 0 (which is empty).
 					if p.pipelineMetadata.NextRound > 0 {
-						p.addMetrics(blkData, importTime)
+						p.addMetrics(blkData, time.Since(start))
 					}
 					// Increment Round
 					p.setError(nil)
 					p.pipelineMetadata.NextRound++
 					p.encodeMetadataToFile()
+					retry = 0
+					p.pipelineMetadata.NextRound++
```

Comment:
Double increment here
```suggestion
					p.setError(nil)
					p.encodeMetadataToFile()
					retry = 0
					p.pipelineMetadata.NextRound++
```